lib/Db/Poll.php

```diff
@@ -191,10 +207,10 @@ public function jsonSerialize(): array {
 			'voteLimit' => $this->getVoteLimit(),
 			'lastInteraction' => $this->getLastInteraction(),
 			'summary' => [
-				'orphanedVotes' => $this->getCurrentUserOrphanedVotes(),
-				'yesByCurrentUser' => $this->getCurrentUserYesVotes(),
-				'countVotes' => $this->getCurrentUserCountVotes(),
 				'userRole' => $this->getUserRole(),
+				'orphanedVotes' => $this->getCurrentUserCountOrphanedVotes(),
+				'yesVotes' => $this->getCurrentUserCountVotesYes(),
```

Comment:
This was renamed from `yesByCurrentUser` to `yesVotes`. Should be reverted or UI needs to be adapted I suppose.

Reply:
Ref: https://github.com/nextcloud/polls/pull/3387/files#diff-13f57c645696e1120c901e44847fa9b40ba79f6aa7f1a29bf1ed86ff4ed70c61R212